app/services/survey_response_processor.rb

```diff
@@ -31,6 +42,14 @@ def update_action
     response
   end
 
+  def delete_existing_action
+    response = Ak::Client.client.delete_action(existing_action_ak_id)
+    if !response.success?
+      raise Error.new("HTTP Response code: #{response.code}, body: #{response.body}")
```

Comment:
Same here as above, I wouldn't mind a bit more verbosity (e.g. that this was an error in deleting existing action)